docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -95,9 +95,9 @@ also maps users in the form ``bob@uk.example.com`` to ``bob_uk``:
 .. literalinclude:: user-mapping.json
     :language: json
 
-Configure this example file to use the :doc:`LDAP </security/ldap>`
-authentication method (which uses the PASSWORD authentication type), as
-follows:
+In your coordinator's ``config.properties`` file, configure this example file to
+use the :doc:`LDAP </security/ldap>` authentication method (which uses the
```

Comment:
```suggestion
In your coordinator's ``config.properties`` file, add the below configuration properties to
use the :doc:`LDAP </security/ldap>` authentication method (which uses the
```

adding a file to a file feels weird and confusing. wdyt about this? (maybe can be worded better).

Reply:
There's two which's. See if you can eliminate one of them